rotkehlchen/db/upgrades/v34_v35.py

```diff
@@ -108,6 +128,30 @@ def _add_blockchain_column_web3_nodes(cursor: 'DBCursor') -> None:
     cursor.execute('DROP TABLE web3_nodes_old')
 
 
+def _rename_assets_in_user_queried_tokens(cursor: 'DBCursor') -> None:
+    """ethereum_accounts_details has the column tokens_list as a json list with identifiers
+    using the _ceth_ formath. Those need to be upgraded to the CAIPS format.
+    """
+    cursor.execute('SELECT account, tokens_list FROM ethereum_accounts_details')
+    update_rows = []
+    for address, token_list in cursor:
+        tokens = json.loads(token_list)
+        new_ids = []
+        for token in tokens.get('tokens', []):
+            new_ids.append(
+                evm_address_to_identifier(
+                    address=token[ETHEREUM_DIRECTIVE_LENGTH:],
+                    chain=ChainID.ETHEREUM,
+                    token_type=EvmTokenKind.ERC20,
+                ),
+            )
+        update_rows.append((json.dumps({'tokens': new_ids}), address))
+    cursor.executemany(
+        'UPDATE ethereum_accounts_details SET tokens_list=? WHERE account=?',
```

Comment:
Question. Shouldn't `ethereum_accounts_details` be transformed as part of going multichain? I mean like yeah, we can put here any token identifiers we want, but I just think that it may be clearer if we include chain in the db schema. Or is it not needed?